build.xml

```diff
@@ -1035,4 +1039,11 @@
       </classpath>
     </eclipse>
   </target>
+
+  <target name="check-forbidden-apis" depends="compile">
+    <fa:forbiddenapis internalRuntimeForbidden="true" classpathref="classpath" dir="${build.dir}">
+      <bundledsignatures name="jdk-unsafe-${javac.version}"/>
```

Comment:
Here is a list for what we can do: [(https://github.com/policeman-tools/forbidden-apis/wiki/BundledSignatures)] On the other hand we can write some custom checks as like: [https://github.com/apache/lucene-solr/blob/master/lucene/tools/forbiddenApis/base.txt]